superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -188,7 +193,7 @@ export default function ActivityTable({
       label: t('Edited'),
       onClick: () => {
         setActiveChild('Edited');
-        setInLocalStorage('activity', { activity: 'Edited' });
+        setInLocalStorage(HOMEPAGE_ACTIVITY_FILTER, { activity: 'Edited' });
```

Comment:
It would be great if we could have an Enum also for the activity types, such as Edited, Created, etc.

Reply:
Yep makes sense!